packages/node_modules/@webex/plugin-meetings/src/stats/metrics.js

```diff
@@ -0,0 +1,112 @@
+import {StatelessWebexPlugin} from '@webex/webex-core';
+
+import {
+  MEETINGS,
+  STATS
+} from '../constants';
+
+/**
+  * Metrics Payload Event
+  * Emitted when a Metric Payload is available
+  * @event metrics:payload
+  * @instance
+  * @type {Object}
+  * @memberof MediaMetrics
+  */
+
+/**
+ * @class MediaMetrics
+ * @private
+ */
+export default class MediaMetrics extends StatelessWebexPlugin {
```

Comment:
Does this need to extend `StatelessWebexPlugin`?

Reply:
it did to access the config, though we could also use the StaticConfig. I am not sure what direction the SDK plugin arch is headed

Reply:
Yes, please do not use the plugin class except for the top level plugin. We are going to need to refactor the meetings plugin to not subclass the plugin throughout their classes.

Reply:
Sure, can change it.